src/Gui/Dialogs/DlgParameterImp.cpp

```diff
@@ -850,14 +850,14 @@ void ParameterValue::onCreateIntItem()
     for (const auto& it : lmap) {
         if (name == QLatin1String(it.first.c_str())) {
             QMessageBox::critical(this,
-                                  tr("Existing item"),
+                                  tr("Existing Item"),
                                   tr("The item '%1' already exists.").arg(name));
             return;
         }
     }
 
     int val = QInputDialog::getInt(this,
-                                   QObject::tr("New integer item"),
+                                   QObject::tr("New Integer Item"),
                                    QObject::tr("Enter number:"),

```

Comment:
```suggestion
                                   QObject::tr("Enter number"),

```

Everywhere else we have: Enter the... here it is just Enter...